samples/SkijaInjectSample/build.gradle.kts

```diff
@@ -64,4 +67,8 @@ tasks.withType<Test> {
         systemProperty("sun.java2d.dpiaware", "false")
         systemProperty("sun.java2d.uiScale", "1")
     }
+}
+
+tasks.withType<KotlinCompile>().configureEach {
+    kotlinOptions.freeCompilerArgs += "-Xopt-in=kotlin.RequiresOptI"
```

Comment:
kotlin.RequiresOptI*n*. Why it is actually needed, just to get rid of warnings?

Reply:
Without it there is a warning:

"This class can only be used with the compiler argument ‘-Xopt-in=kotlin.RequiresOptIn’"

https://kotlinlang.org/docs/reference/opt-in-requirements.html#experimental-status-of-the-opt-in-requirements

Reply:
OK, just put the proper opt-in name then.

Reply:
Oh, thanks. I didn't notice that I lost letter in the commit :)